### PR TITLE
[FormBundle] Order the FormSubmissionFields like their respective PageParts on the FormPage

### PR DESCRIPTION
**Pull request by kln3wrld**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | yes
| BC breaks?    | yes
| Deprecations? | no
| Fixed tickets | [445](https://github.com/Kunstmaan/KunstmaanBundlesCMS/issues/445)

### CODE REVIEW

File: src/Kunstmaan/FormBundle/Controller/FormSubmissionsController.php [hunk 1]
```diff
@@ -89,11 +89,16 @@ public function listAction(Request $request, $nodeTranslationId)
      */
     public function editAction($nodeTranslationId, $submissionId)
     {
-        $em              = $this->getDoctrine()->getManager();
-        $nodeTranslation = $em->getRepository('KunstmaanNodeBundle:NodeTranslation')->find($nodeTranslationId);
-        $formSubmission  = $em->getRepository('KunstmaanFormBundle:FormSubmission')->find($submissionId);
+        $em                   = $this->getDoctrine()->getManager();
+        $nodeTranslation      = $em->getRepository('KunstmaanNodeBundle:NodeTranslation')->find($nodeTranslationId);
+        $formSubmission       = $em->getRepository('KunstmaanFormBundle:FormSubmission')->find($submissionId);
+        $formSubmissionFields = $em->getRepository('KunstmaanFormBundle:FormSubmissionField')->getFieldsOrderedBySequence($formSubmission);
```
Comment (wimvds) [inline src/Kunstmaan/FormBundle/Controller/FormSubmissionsController.php]: Why would you fetch them separately? The fields are a property of FormSubmission, and can be sorted there according to their sequence by adding a simple Doctrine annotation. cfr. http://doctrine-orm.readthedocs.org/en/latest/tutorials/ordered-associations.html

Comment (kln3wrld) [inline src/Kunstmaan/FormBundle/Controller/FormSubmissionsController.php]: Aha! That's indeed what I needed but didn't find right away. Will fix tomorrow. Thx!

**Comment by roderik** [comment]
@kln3wrld please squash

**Comment by roderik** [comment]
@kln3wrld put the upgrade info in upgrade-3.2, not in upgrade. Then I can merge it

**Pull request merged**